### ISSUE TITLE
[Improve] [Feedback] User cannot copy email which already inputted in "to" "cc" "bcc" fields

### ISSUE THREAD

**Issue by QuangHoang1210** [linagora/tmail-flutter#2488]
1, Feedback from Anton:
`It also would have been great if I was able to copy emails from "to", "cc" fields, right not it's absolutely impossible `

-> We have drag and drop email address now but we will improve copy action. 
_Suggest_: Email address will copied if user click on email address tag at "to", "cc", "bcc" field

**Video drag and drop:**

https://github.com/linagora/tmail-flutter/assets/124866146/17b7332f-6c92-4366-8d1e-860159caa76b

**Comment by chibenwa** [comment]
What gmail is doing?

**Comment by chibenwa** [comment]
Answer:

When I click an email address in the composer, it displays contact infos:

![image](https://github.com/linagora/tmail-flutter/assets/6928740/550df0a3-074f-44c9-aac5-91390d2523e3)


So in tmail we could reuse recipient pop up?

![image](https://github.com/linagora/tmail-flutter/assets/6928740/47d22606-350b-40bb-8664-7417c3ad18cb)


What do you think @tprudentova @shepilov  @hoangdat  ?

**Comment by shepilov** [comment]
> Answer:
> 
> When I click an email address in the composer, it displays contact infos:
> 
> ![image](https://private-user-images.githubusercontent.com/6928740/297670861-550df0a3-074f-44c9-aac5-91390d2523e3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU1Njc5MjgsIm5iZiI6MTcwNTU2NzYyOCwicGF0aCI6Ii82OTI4NzQwLzI5NzY3MDg2MS01NTBkZjBhMy0wNzRmLTQ0YzktYWFjNS05MTM5MGQyNTIzZTMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDExOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMThUMDg0NzA4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2I5ZjA5MDZkNjY3NjFkZGY3NTdmMjFlZTVmNmJkNDhjZjQ2N2NhMmI5MDBkNjViZDI1ZjZmZTYxMjhhZGY3MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.lc73wYLr-cx4v1IjwJmriHcU3zU2NR3iMYnKe1s0ANM)

Yes, it's fine, but it doesn't solve the problem when I need to copy all list of the users and paste it to another email.

It could also be for example this
<img width="410" alt="image" src="https://github.com/linagora/tmail-flutter/assets/14924963/1d2724ca-fd40-4647-b690-00e276f23f33">
when all the info is displayed, and I can copy a list of users and paste them into another email

**Comment by chibenwa** [comment]
You mean 

https://github.com/linagora/tmail-flutter/assets/6928740/682c77bd-6a74-42f5-b595-77f20e6132b7

? In addition to what I described above?

**Comment by shepilov** [comment]
@chibenwa exactly

**Comment by chibenwa** [comment]
@QuangHoang1210 can you split this in two tickets?

**Comment by chibenwa** [comment]
Exist in other tickets
